themes/sensei-course-theme/templates/modern/class-sensei-lm-template-modern.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * The class file for Sensei_LM_Template_Modern.
+ *
+ * @author      Automattic
+ * @package     Sensei
+ * @version     $$next-version$$
+ */
+
+/**
+ * Class Sensei_LM_Template_Modern
+ */
+class Sensei_LM_Template_Modern {
+	/**
+	 * The version of this template.
+	 *
+	 * @var string
+	 */
+	const VERSION = '1.0.0';
+
+	/**
+	 * The name of the template.
+	 *
+	 * @var string
+	 */
+	const NAME = 'modern';
+
+	/**
+	 * Returns the template info.
+	 */
+	public static function get_info() {
+		$base_url = Sensei_Course_Theme::instance()->get_course_theme_root_url() . '/templates';
+		$name     = self::NAME;
+
+		return [
+			'name'        => $name,
+			'title'       => __( 'Modern', 'sensei-lms' ),
+			'content'     => [
+				'lesson' => '',
+				'quiz'   => '',
+			],
+			'version'     => self::VERSION,
+			'styles'      => [],
+			'scripts'     => [],
+			'screenshots' => [
+				'thumbnail' => "$base_url/$name/img-thumb.jpg",
+				'full'      => "$base_url/$name/img-full.jpg",
```

Comment:
One option we can think about for moving the big files out of the plugin is using external URLs, hosting them on senseilms.com. Or could also decrease the jpeg quality a bit more.

Reply:
> One option we can think about for moving the big files out of the plugin is using external URLs, hosting them on senseilms.com.

Interesting. We could do this. I am a bit worried that it could be easily broken if some url behavior changes on the senseilms.com.


> Or could also decrease the jpeg quality a bit more.

I tried to make them smaller, but then the image gets a bit blurry. I think we want the images to look sharp, otherwise it would be hard to sell the premium templates.

I also tried to make them png, but it gets bigger. The problem is that it has pictures in them. Pictures weight a lot in png. If we could use only vector images, then png images could be much smaller.

Reply:
Yeah that was my other thought too, we can replace the pictures with some very simple illustration.